src/tools/DoubleTapFitToWindowTool.test.js

```diff
@@ -7,6 +7,10 @@ jest.mock('../externalModules.js', () => ({
   }
 }));
 
+jest.mock('../import.js', () => ({
+  default: jest.fn()
+}));
```

Comment:
There are ways to do this globally as part of a jest setup, and then override that behavior in an individual test suite if you don't want the default